Modules/Relationships/test/Relationships.Domain.Tests/Tests/RelationshipTests.cs

```diff
@@ -50,6 +52,20 @@ public void New_Relationship_Has_Correct_Data()
         change.Response.Should().BeNull();
     }
 
+    [Fact]
+    public void Raises_RelationshipChangeCreatedDomainEvent_when_creating()
+    {
+        // Act
+        var relationship = CreatePendingRelationship();
+        var change = relationship.Changes.GetOpenCreation()!;
```

Comment:
This isn't part of `Act`.

Reply:
But what would be part of `Act` (since the domain event is raised in the constructor of `RelationshipChange` and therefore `Relationship`)?

Reply:
The constructor call, of course.